### PR TITLE
Create Rake Task

### LINKED ISSUE #7: Create rake task to import CSV file

### PR DESCRIPTION
**Pull request by smainar**
## Pull Request Review Template

- [ ] Wrote Tests
- [X] Implemented
- [ ] Reviewed

### Necessary checkmarks:
- [X] All Tests are Passing
- [X] The code will run locally

### Type of change
- [X] New feature
- [ ] Bug Fix

### Implements/Fixes:
- Create rake task to import historical data from 2016 Summer Olympics

### Description of Changes:
- Add historical data from 2016 Summer Olympics
- Add rake task to import historical data from 2016 Summer Olympics

closes #7 

### Check the correct boxes
- [X] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

### Testing Changes
- COVERAGE SCORE: 100%
- [ ] New Tests have been added
- [X] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed (Please describe what in the world happened)

### Checklist:
- [X] My code has no unused/commented out code
- [X] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have fully tested my code

### CODE REVIEW

**Comment by smainar** [comment]
@Myrdden - This PR contains my rake task for importing historical data from the 2016 Summer Olympics. I'd appreciate your review. Please let me know if you have any suggestions for refactoring.

My rake task loaded the following:
- 197 teams
- 34 sports
- 2856 Olympians
- 305 events
- 3485 Olympian events

Do the numbers above match what you have?

File: lib/tasks/import.rake [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  if 'NA'
+    0
+  elsif 'Gold'
+    1
+  elsif 'Silver'
+    2
+  else
+    3
+  end
+end
```
Comment (Myrdden) [inline lib/tasks/import.rake:62]: Also could just be like
```
medals = ['NA', 'Gold', 'Silver', 'Bronze']
medals.find_index(row[:medal])
```

**Review by Myrdden — COMMENTED** [review]
Didn't occur to me to use find_or_create when I was doing this lol. I thought it would take a really long time because it makes like 5 queries per line but evidently it's not so bad.

👌

**Comment by Myrdden** [comment]
> 
> 
> @Myrdden - This PR contains my rake task for importing historical data from the 2016 Summer Olympics. I'd appreciate your review. Please let me know if you have any suggestions for refactoring.
> 
> My rake task loaded the following:
> 
>     * 197 teams
> 
>     * 34 sports
> 
>     * 2856 Olympians
> 
>     * 305 events
> 
>     * 3485 Olympian events
> 
> 
> Do the numbers above match what you have?

yee

File: lib/tasks/import.rake (hunk 1 above)
Comment (smainar) [reply lib/tasks/import.rake:62]: @Myrdden - Thank you for that suggestion on the refactor! I hadn't thought of of using the `find_index` method.

**Pull request merged**